xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -271,6 +271,7 @@ void CGUIWindowVideoBase::OnItemInfo(CFileItem* pItem, ADDON::ScraperPtr& scrape
     Refresh();
     m_viewControl.SetSelectedItem(itemNumber);
   }
+  return modified;
```

Comment:
Returning true is not working since `ShowIMDB()` fails. I think the variable name `modified` is just badly chosen here. In case it fails we're unable to show information and it should just play.